modules/local/append_profiles/main.nf

```diff
@@ -0,0 +1,20 @@
+process APPEND_PROFILES {
+    tag "Append additional reference profiles"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/csvtk:0.22.0--h9ee0642_1' :
+        'biocontainers/csvtk:0.22.0--h9ee0642_1' }"
+
+    input:
+    path(reference_profiles)
+    path(additional_profiles)
+
+    output:
+    path("*.tsv")
+
+    script:
+    """
+    csvtk concat -t ${reference_profiles} ${additional_profiles} > profiles_ref.tsv
```

Comment:
Does `csvtk concat` make sure the columns all match (I forgot if it does)? What would happen if there are different loci in the `reference_profiles` and `additional_profiles`? I wonder if there's a way for us to check that?

Also, is there some way for us to check for e.g., an overlap of sample identifer in the `reference_profiles` and `additional_profiles`? Doesn't have to be in this process, but just something else I thought of for validation of the data.

Reply:
No, `csvtk` does not have a way to ensure all columns match.
Instead I added a function and a check to the script in both `append_clusters` and `append_profiles` here: [8ae527f](https://github.com/phac-nml/gasnomenclature/pull/29/commits/8ae527f185c3e89de9e5f9b2d7e0b8ca88ea6356)
I also used `csvtk uniq` to ensure there are no overlapping `sample_id's` : [8ae527f](https://github.com/phac-nml/gasnomenclature/pull/29/commits/8ae527f185c3e89de9e5f9b2d7e0b8ca88ea6356)

Reply:
Thanks so much Kyla. This is amazing work 😄